cmake/OpenCL.cmake

```diff
@@ -1,30 +1,19 @@
 option (ENABLE_OPENCL "Use OpenCL optimizations" ON)
 
 set (OPENCL_AMDSDK /opt/AMDAPPSDK-3.0 CACHE PATH "Installation path for the installed AMD OpenCL SDK, if used")
+set (OPENCL_NVIDIASDK "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v10.0" "Installation path for the installed NVIDIA CUDA Toolkit, if used")
```

Comment:
Tag by OS, namely, `OPENCL_WIN_NVIDIASDK`.

Reply:
It's not necessary, since `find_package()` will handle this if it's installed, but I'll leave a conditional check for AMD on VMWare.

Reply:
Done.